src/routes/landing/components/CitySelector.js

```diff
@@ -76,10 +122,45 @@ class CitySelector extends React.PureComponent<PropsType> {
     }, [])
   }
 
+  renderListByLocation (): React.Node {
+    if (this.props.currentLatitude === null || this.props.currentLongitude === null) {
+      return null
+    }
+    const currentLatitude = this.props.currentLatitude
+    const currentLongitude = this.props.currentLongitude
+    let cities = this.props.cities.filter(_city => _city.live)
+      .sort((a: CityModel, b: CityModel) => compareDistance(a, b, currentLongitude, currentLatitude))
+    const numberOfClosestCities = 3
+    cities = cities.slice(0, numberOfClosestCities)
+    const maximalDistance = 90
+    cities = cities.filter(_city => currentDistance(_city, currentLongitude, currentLatitude) < maximalDistance)
```

Comment:
It would be nicer to define `cities` as a `const` variable right away by chaining the array operations like so: 
```js
    const numberOfClosestCities = 3
    const maximalDistance = 90
    const cities = this.props.cities
      .filter(_city => _city.live)
      .sort((a: CityModel, b: CityModel) => compareDistance(a, b, currentLongitude, currentLatitude))
      .slice(0, numberOfClosestCities)
      .filter(_city => calculateDistanceToCity(_city, currentLongitude, currentLatitude) < maximalDistance)
```